packages/manager/apps/dedicated/package.json

```diff
@@ -40,10 +40,12 @@
     "@ovh-ux/manager-models": "^1.0.0",
     "@ovh-ux/manager-nasha": "^1.0.0 || ^2.0.0",
     "@ovh-ux/manager-navbar": "^4.0.0 || ^5.0.0",
+    "@ovh-ux/manager-storage": "^0.0.0 || ^1.0.0",
     "@ovh-ux/manager-ng-layout-helpers": "^1.0.1 || ^2.0.0",
     "@ovh-ux/manager-notifications-sidebar": "^0.0.0 || ^1.0.0",
     "@ovh-ux/manager-preloader": "^1.0.0",
     "@ovh-ux/manager-server-sidebar": "^2.0.0 || ^3.0.0",
+    "@ovh-ux/manager-storage": "^0.0.0 || ^1.0.0",
```

Comment:
No errors declaring the same package twice ?

Reply:
it's a merge mistake. I correct that.